src/modules/auth/auth.service.ts

```diff
@@ -127,20 +139,14 @@ export class AuthService {
         refreshToken,
       };
     }
```

Comment:
what is it, its service . service should accept only param that should be used. no - body: GoogleAuthSingUpDto, instead => email, name, jti

Reply:
I understand the remark. Instead of a badi, I will transfer a token. I can’t pass {email, name, jti} because I think I'm not allowed to decode the token at the controller level.